src/css/our_collection.css

```diff
@@ -0,0 +1,133 @@
+.collection {
+  background-color: #1c2225;
+  padding-bottom: 80px;
+  background-image: url(../img/our_collection/mobile/emblem2-mob.png);
+  background-repeat: no-repeat;
+  background-position-x: left;
+  background-position-y: 298px;
+  background-size: 72px 177px;
+}
+
+@media (min-device-pixel-ratio: 2),
+  (min-resolution: 192dpi),
+  (min-resolution: 2dppx) {
+  .collection {
+    background-image: url(../img/our_collection/mobile/emblem2-mob-2x.png);
+  }
+}
+@media screen and (min-width: 768px) {
+  .collection {
+    padding-bottom: 100px;
+    background-position-y: 221px;
+    background-image: url(../img/our_collection/tablet/emblem2-tab.png);
+    background-size: 100px;
+  }
+  @media (min-device-pixel-ratio: 2),
+    (min-resolution: 192dpi),
+    (min-resolution: 2dppx) {
+    .collection {
+      background-image: url(../img/our_collection/tablet/emblem2-tab-2x.png);
+    }
+  }
+}
+
+@media screen and (min-width: 1280px) {
+  .collection {
+    padding-bottom: 150px;
+    padding-top: 70px;
+    background-image: url(../img/our_collection/desctop/emblem2-desc-2x.png);
+    background-size: 150px;
+  }
+  @media (min-device-pixel-ratio: 2),
+    (min-resolution: 192dpi),
+    (min-resolution: 2dppx) {
+    .collection {
+      background-image: url(../img/our_collection/desctop/emblem2-desc-2x.png);
+    }
+  }
+}
+
+.collection-title {
+  text-align: center;
+  color: #fcf9f9;
+  margin-bottom: 40px;
+}
+@media screen and (min-width: 768px) {
+  .collection-title {
+    line-height: 1.45;
+    font-size: 44px;
+    letter-spacing: -0.03em;
+  }
+}
+.collection-item {
+  width: auto;
+  height: auto;
+  margin: 0;
+}
+@media screen and (min-width: 768px) {
+  .collection-item {
+    width: calc((100% - 20px) / 2);
+  }
+}
+@media screen and (min-width: 1280px) {
+  .collection-item {
+    width: calc((100% - 40px) / 3);
+  }
+}
+.collection-name {
+  font-size: 14px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  font-weight: 500;
+  line-height: 1;
+  letter-spacing: 0.05em;
+  text-transform: uppercase;
+  margin-bottom: 8px;
+  white-space: nowrap;
+}
+@media screen and (min-width: 768px){
+.collection-name{
+  font-size: 18px;
+}
+}
+
+.collection-list {
+  gap: 20px;
+  justify-content: center;
+  row-gap: 20px;
+}
+@media screen and (min-width: 1280px) {
+  .collection-list {
+    row-gap: 40px;
+  }
+}
+.collection-price {
+  line-height: 1;
+  letter-spacing: 0.05em;
+}
+@media screen and (min-width: 768px){
+  .collection-price{
+    font-size: 18px;
+  }
+  }
+.collection-photo {
+  border-radius: 4px;
+}
+@media screen and (min-width: 375px) {
+  .collection-photo {
+    width: 335px;
+  }
+}
+@media screen and (min-width: 768px) {
+  .collection-photo {
+    width: 342px;
+  }
+}
+@media screen and (min-width: 1280px) {
+  .collection-photo {
+    width: 347px;
+  }
+}
+.collection-information {
+  margin-top: 18px;
+}
```

Comment:
@media screen and (min-width: 768px) {
.collection-photo {
    width: 342px;
  }
.collection-price{
    font-size: 18px;
  }
.collection-name{
  font-size: 18px;
}
.collection-item {
    width: calc((100% - 20px) / 2);
  }
.collection-title {
    line-height: 1.45;
    font-size: 44px;
    letter-spacing: -0.03em;
  }
.collection {
    padding-bottom: 100px;
    background-position-y: 221px;
    background-image: url(../img/our_collection/tablet/emblem2-tab.png);
    background-size: 100px;
  }
}